README.md

```diff
@@ -14,18 +14,59 @@ This solves many use cases like:
 
 # Features
 
-  - A side view to display product hierarchy.
+  - A left pane to display project hierarchy.
   - Search functionality for faster access to VIs.
-  - Access to support for multi frame structures like case structures, event structures, disable structures and stacked structures.
+  - Support for viewing Multi Frame Structures like Case Structures, Event Structures, Disable Structures and Stacked Structures.
   - Navigation to SubVIs and Dynamic Dispatch SubVIs.
-  - Context help for SubVIs and most of the function nodes.
+  - Context help for SubVIs and most of the primitive nodes.
+
+Note: FINALE is not supported and this is mostly internal tooling that we are exposing. This is more of a 'hack' that we built, this is not polished code/design.
 
 # Setting up FINALE for your code
 
-The web app is developed using npm package manager. Run the following commands to produce a built deployment under a 'build' directory.
+FINALE has two parts, the HTML Generator and the WebApp. The HTML Generator converts G-Code to metadata in the form of JSONs, images etc. These are input to the WebApp which opens a web-based viewer for the files converted. 
+## Prerequisites: 
+- LabVIEW
+- Browser: Google Chrome 
```

Comment:
Just Chrome?

Reply:
Added Firefox as well, and mentioned that Edge is not fully supported.